browser/brave_shields/android/java/src/org/chromium/chrome/browser/brave_shields/brave_first_party_storage_clean_animation.json

```diff
@@ -0,0 +1,1869 @@
+{
```

Comment:
it would be good to share that json with iOS `ios/brave-ios/Sources/Brave/Assets/Lottie/shred.json`. As they are identical. We would need to change only one in that case if it ever changes in the future. Let's create a follow up on that.

Reply:
https://github.com/brave/brave-browser/issues/51556